local-cli/rnpm/windows/src/windows.js

```diff
@@ -25,7 +25,7 @@ const REACT_NATIVE_WINDOWS_GENERATE_PATH = function() {
 };
 
 module.exports = function windows(config, args, options) {
-  const name = args[0] ? args[0] : Common.getReactNativeAppName();
+  const name = config[0] ? config[0] : Common.getReactNativeAppName();
```

Comment:
args is an object not an array, and when you run`react-native windows awesomeproject`, then `configs[0] = awesomeproject`. This has been changed to reflect this